app/views/content/layout/_navbar.html.erb

```diff
@@ -25,7 +25,7 @@
       <%- end %>
 
       <%= active_link_to "About", about_path, class: "nav-item nav-link" %>
-      <%= active_link_to "Papers", papers_path, class: "nav-item nav-link" %>
+      <%= active_link_to "Papers", published_papers_path, class: "nav-item nav-link" %>
       <%= active_link_to "Docs", "https://joss.readthedocs.io/en/latest/index.html", target: '_blank', class: "nav-item nav-link" %>
```

Comment:
I was going to add this change to `views/papers/show.html.erb` to keep the top links behaving the same way everywhere but that raised another question: why is the paper' show view not using the navbar partial? 
Just in case I'm missing something I've issued this change as a separate [PR against the fresh branch here](https://github.com/openjournals/joss/pull/555).

Reply:
Thanks for that PR @xuanxu!